### ISSUE TITLE
Dynamic system for Flag

### ISSUE THREAD

**Issue by olivierd1** [moloch--/RootTheBox#468]
Hi! Is it planned to be able to set the dynamic scoring system for each flag instead of a global configuration ?

**Comment by jeffg2k** [comment]
No, haven't had a request for that.  Might be a bit confusing.  What is the use case?

**Comment by olivierd1** [comment]
The goal is to be able to have some "special flags" among all the other ones. Not a high priority but if you don't know what to add... ;)
